cmd/server/entrypoint.go

```diff
@@ -83,27 +117,39 @@ func StartProxySvr(cliCtx *cli.Context) error {
 			}
 		}()
 		log.Info("started metrics server", "addr", svr.Addr())
-		m.RecordUp()
+		metrics.RecordUp()
 	}
 
 	return ctxinterrupt.Wait(cliCtx.Context)
 }
 
 // TODO: we should probably just change EdaClientConfig struct definition in eigenda-client
 func prettyPrintConfig(cliCtx *cli.Context, log logging.Logger) error {
+	redacted := "******"
+
 	// we read a new config which we modify to hide private info in order to log the rest
-	cfg := server.ReadCLIConfig(cliCtx)
-	if cfg.EigenDAConfig.EdaClientConfig.SignerPrivateKeyHex != "" {
-		cfg.EigenDAConfig.EdaClientConfig.SignerPrivateKeyHex = "*****" // marshaling defined in client config
+	cfg, err := config.ReadCLIConfig(cliCtx)
```

Comment:
Nit - can change function name to 'ReadFromCLI' since referenced from a config package

Reply:
It's possible that packages will shift when [this issue](https://linear.app/eigenlabs/issue/DAINT-327/cleanup-v2-proxy-config-code) is worked, let's leave as-is for now, and address during the cleanup